sentry-android-core/src/main/java/io/sentry/android/core/AndroidMemoryCollector.java

```diff
@@ -0,0 +1,17 @@
+package io.sentry.android.core;
+
+import android.os.Debug;
+import io.sentry.IMemoryCollector;
+import io.sentry.MemoryCollectionData;
+import org.jetbrains.annotations.ApiStatus;
+
+@ApiStatus.Internal
+public class AndroidMemoryCollector implements IMemoryCollector {
+  @Override
+  public MemoryCollectionData collect() {
+    long now = System.currentTimeMillis();
```

Comment:
Is `System.currentTimeMillis()` good enough as timestamp? Or should I go with `DateUtils.getCurrentDateTime().getTime()`?